docs/content/sips/017-configuring-and-emitting-observability.md

```diff
@@ -31,34 +31,32 @@ This proposal assumes that all observability data produced should be OTEL compli
 
 ### Configuring observability
 
-The developer must be able to configure the endpoints of OTLP compliant collectors where traces and metrics can be sent. This can be expressed in the `spin.toml` manifest.
+The developer must be able to configure the endpoints of OTLP compliant collectors where traces and metrics can be sent. This can be expressed in the `runtime-config.toml`.
 
 ```toml
-[application]
-otlp_grpc_traces_endpoint = "http://localhost:4317"
-otlp_grpc_metrics_endpoint = "http://localhost:4317"
+[observability.tracing]
+endpoint = "http://localhost:4317"
+protocol = "http/protobuf" # "grpc" or "http/protobuf" or "http/json" and it defaults to "http/protobuf"
+[observability.metrics]
+endpoint = "http://localhost:4317"
+protocol = "http/protobuf" # "grpc" or "http/protobuf" or "http/json" and it defaults to "http/protobuf"
 ```
 
-**INPUT NEEDED:** How do we want to configure these endpoints? Where do they belong in the manifest? Do we want to allow for separate endpoints for traces and metrics? Do we want to allow for choosing between gRPC and HTTP? Should we let them pass a flag to `spin` to override the manifest value?
+**BIKESHED:** `observability.tracing`, `otel.tracing`, `telemetry.tracing`....
```

Comment:
Maybe something like:

```toml
[telemetry.otlp]  # OTLP is the protocol name
endpoint = "..."
traces = true
metrics = true  # default false? 🤷
```

Reply:
Did you purposely remove the ability to configure the protocol?

Reply:
No, I was just trying to capture an example that covered the protocol name and toggling e.g. traces vs metrics.